inventory/quay-enterprise/group_vars/database.yml

```diff
@@ -1,9 +1,18 @@
 ---
 
-# Database Configuration
-database_type: postgresql
-database_name: quay
-database_username: quay
-database_password: quay
-database_admin_username: root
-database_admin_password: quayadmin
\ No newline at end of file
+# Quay Database Configuration
```

Comment:
Suggestion: Can we make these dictionaries instead of prefix'ing all the variables? e.g.:
```
quay:
  database_type: ...
  database_name: ...
```

We have started to do this for other areas within `infra-ansible`, so would be good to see the same here unless you disagree?